frame/oracle/src/lib.rs

```diff
@@ -380,35 +382,40 @@ pub mod pallet {
 		}
 	}
 
-	impl<T: Config> Oracle for Pallet<T> {
+	impl<T: Config> Oracle for Pallet<T>
+	where
```

Comment:
This bound can be removed as block numbers can be reliably `unique_saturated_into` u128 where necessary.